# Message file for nodes in map

uint8 mapPointID
geometry_msgs/Point position
float64 angleOfApproach
